www_src/pages/page/page.less

```diff
@@ -38,30 +38,41 @@
     }
   }
 
-  .overlay {
+  .overlay-enter {
     position: fixed;
     top: 0;
     left: 0;
     height: 100%;
     width: 100%;
-    z-index: 998;
+    z-index: 999;
     background: #1A2237;
     opacity: 0;
     visibility: hidden;
-    transition: all 0.1s ease-in-out;
-    &.active {
+    transition: all 1s ease-in-out;
+
+    &.overlay-enter-active {
       visibility: visible;
       opacity: 0.7;
     }
   }
 
+  /* Necessary to force-remove ReactCSSTransitionGroup elements in unminified React: */
+  .overlay-leave {
+    opacity: 1;
+    transition: opacity .5s ease-in;
+    &.overlay-leave-active {
+      opacity: 0.01;
+    }
+  }
+  /* See the end of https://facebook.github.io/react/docs/animation.html#getting-started */
+
   .controls {
     @_transformDistance: 55px;
     position: fixed;
     bottom: 20px;
     left: 0;
     right: 0;
-    z-index: 999;
+    z-index: 998;
```

Comment:
and then we can change this one to 2147483646, with a comment to explain it has to be "the highest - 1" so that the overlay still wins.